scipy/spatial/distance.py

```diff
@@ -1573,8 +1591,20 @@ def russellrao(u, v, w=None):
     return float(n - ntt) / n
 
 
+def _deprecate_sokalmichener(fn):
+    @wraps(fn)
+    def wrapper(*args, **kwargs):
+        warnings.warn(
+            "The sokalmichener metric is deprecated since Scipy 1.15.0 and will "
+            "be removed in SciPy 1.17.0.", DeprecationWarning, stacklevel=2)
```

Comment:
should we mention the metric folks should use instead?

Reply:
Good suggestion. Let me add that too!